lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -14,34 +14,34 @@ module GitHelper
       # @return [Bool] True if the current directory is the root of a git repo
       #         (i.e. a local working copy) or a subdirectory of one.
       #
-      def self.is_git_repo?(path: nil)
-        args = ['git']
-
-        unless path.nil?
-          # `path` might represent a file that doesn't yet exist but is in a
-          # valid Git repo. In such case, the `git rev-parse` we're currently
-          # building would fail. To avoid that, let's perform the check on the
-          # first existing ancestor of the given path – which will be the path
-          # itself if it exists.
-          path = first_existing_ancestor_of(path: path)
-          args += ['-C', File.file?(path) ? File.dirname(path) : path]
-        end
-
-        args += ['rev-parse']
-
-        Action.sh(args, print_command_output: false) do |status, _, _|
-          status.success?
-        end
+      def self.is_git_repo?(path: Dir.pwd)
```

Comment:
🤦 

Why did I have a if `nil?` default to `Dir.pwd` check in the original version where I could have defaulted to `Dir.pwd` in the first place?!